saleor/settings.py

```diff
@@ -113,13 +113,21 @@ def get_url_from_env(name, *, schemes=None) -> str | None:
 # This variable should be set to `replica`
 DATABASE_CONNECTION_REPLICA_NAME = "replica"
 
+# If replica env is not set, then always try to use the default env first.
+# If no envs are set, then the `default` value will be used.
+# This allows us to be compatible with previous flows where only single
+# env was used.
+REPLICA_ENV_NAME = "DATABASE_URL_REPLICA"
```

Comment:
Can we change the name to be more precise with this one, e.g: `DATABASE_URL_REPLICA_ENV_NAME`.